llama_index/readers/base.py

```diff
@@ -1,10 +1,14 @@
 """Base reader class."""
 from abc import abstractmethod
-from typing import Any, List
+from typing import Any, Dict, List
 
-from llama_index.bridge.langchain import Document as LCDocument
+try:
+    from pydantic.v1 import Field
+except ImportError:
+    from pydantic import Field
```

Comment:
Made a PR to consolidate all the try/except logic into a single bridge file: https://github.com/jerryjliu/llama_index/pull/7502

Reply:
Thank you! I've been meaning to do that lol